### ISSUE TITLE
In-depth tutorial how to drop-in to replace official Angular

### ISSUE THREAD

**Issue by truongsinh** [fergaldoyle/angular.js-ie8-builds#16]
Hi, my colleagues write a blog post how to drop-in to replace official Angular, because of path difference: http://smarp.smh.re/0GY .Hope you find it beneficial and update the README accordingly.

**Comment by henns20** [comment]
@truongsinh - checked out your tut and couldn't get it to work. 

Made changes - works still with 1e9+ using browserstack. I am using standard angular router to serve up route views pages of the site - using ng-view element/attribute in the index.php

Don't know how to completely debug. In this scenario.
Didn't know if you had any similar issues or understand what could be the problem. 


Attached image of the error messages - 
![ie-8bug](https://cloud.githubusercontent.com/assets/6235104/8510101/91a9d128-229e-11e5-8964-e4d088e5eeaf.png)

**Comment by truongsinh** [comment]
Hmm, do you have your work online somewhere online so we can debug ourselves?

**Comment by henns20** [comment]
Yep on a development url at http://frudea.com/   thanks. I have to revisit it myself - needs a little cleaning up and one more go.

I just couldn't  devote a lot of hours to it - I was hoping a for a straightforward fix - but besides your tut - docs are limited.

**Comment by henns20** [comment]
I could also push it to github

**Comment by henns20** [comment]
pushed it to github in the mean time it. My use case or need is basic routing - really. In ie 8 I just get a page that indicates the router is not working. 
https://github.com/henns20/ironview-angular-ie8-build-version/tree/master/frudea
